### PR TITLE
renamed files and folders to fit new repo name

### PR DESCRIPTION
**Pull request by am-kaiser**
Changed names of files and folder to fit the new name of this repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
+Its core functionality is based on the [IOOS compliance checker](https://github.com/ioos/compliance-checker). The AtMoDat Standard Compliance Checks library makes use of [cc-yaml](https://github.com/cedadev/cc-yaml) which provides a plugin for the [IOOS compliance checker](https://github.com/ioos/compliance-checker) that generates check suites from YAML descriptions. Furthermore, the [Compliance Check Library](https://github.com/cedadev/compliance-check-lib) is used as the basis to define generic, reusable compliance checks. This repository is an extension of this library as it holds specific checks to ensure compliance with the AtMoDat Standard. Moreover, it contains checks to verify the compliance to the [AtMoDat CVs](https://github.com/AtMoDat/AtMoDat_CVs).
```
Comment (jkretz) [inline README.md:5]:
```suggestion
Its core functionality is based on the [IOOS compliance checker](https://github.com/ioos/compliance-checker). The AtMoDat Standard Compliance Checks library makes use of [cc-yaml](https://github.com/cedadev/cc-yaml), which provides a plugin for the [IOOS compliance checker](https://github.com/ioos/compliance-checker) that generates check suites from YAML descriptions. Furthermore, the [Compliance Check Library](https://github.com/cedadev/compliance-check-lib) is used as the basis to define generic, reusable compliance checks. This repository is an extension of this library as it holds specific checks to ensure compliance with the AtMoDat Standard. Moreover, it contains checks to verify the compliance to the [AtMoDat CVs](https://github.com/AtMoDat/AtMoDat_CVs).
```

**Review by jkretz — CHANGES_REQUESTED** [review]
LGTM 
Just a comma is missing

**Review by jkretz — APPROVED** [review]

**Comment by jkretz** [comment]
I am about to merge this branch into the master. Before doing so, I want to unify the run_checks.bash and run_checks_folder.bash

**Comment by am-kaiser** [comment]
> 
> 
> I am about to merge this branch into the master. Before doing so, I want to unify the run_checks.bash and run_checks_folder.bash

Thanks!

**Comment by jkretz** [comment]
I replaced `run_checks.bash` and `run_checks_folder.bash` with `run_checks.py`. This python scirpt can also be used to quickly summarize the results of the respective checker in the future. Furthermore, this script can also be used to convert the `*.txt` from the cfchecker into a `*.json` in the future as well.

**Pull request merged**